packages/core/src/codewhisperer/service/transformByQ/transformApiHandler.ts

```diff
@@ -56,6 +56,18 @@ export function throwIfCancelled() {
     }
 }
 
+export async function updateJobHistory() {
```

Comment:
Why is this function async?

Reply:
Yeah this one has no need to be, can change it back.